pkg/plugin/manager.go

```diff
@@ -52,6 +55,12 @@ func NewManager() (*Manager, error) {
 		return manager, fmt.Errorf("while running all plugin: %w", err)
 	}
 
+	// Install plugin wallet
+	err = manager.InstallWallet()
```

Comment:
Shouldn't we check if the plugin is already installed before trying to install it ?